modules/logger/src/lib.rs

```diff
@@ -73,10 +73,41 @@ impl Logger {
                 let time = DateTime::<Utc>::from(header.timestamp);
                 let image = &header.image;
                 let pid = &header.pid;
-                let payload = &event.payload();
+                let payload = event.payload();
 
-                println!("[{time} \x1b[1;30;43mTHREAT\x1b[0m {image} ({pid})] [{source} - {info}] {payload}")
+                terminal::print_threat(&time, image, *pid, source, info, payload);
```

Comment:
(style) I don't particularly like the deconstructing of Event just to forward it to a function. It's verbose, it's repeated 3 times and with no added value.

I would make a single `terminal::print_event(event: &Event)` which checks if it's a threat or not. `process` would call it only if `event.header.threat.is_some()`.